launcher/minecraft/mod/TexturePackParse_test.cpp

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-3.0-only
+/*
+ *  PolyMC - Minecraft Launcher
+ *  Copyright (c) 2022 flowln <flowlnlnln@gmail.com>
```

Comment:
imagine the confusion of people looking at the git blame, when i never ever touched this file :P